packaging/docker/custom/ubuntu.Dockerfile

```diff
@@ -0,0 +1,37 @@
+ARG GRAFANA_VERSION="latest-ubuntu"
+
+FROM grafana/grafana:${GRAFANA_VERSION}-ubuntu
+
+USER root
+
+ARG GF_INSTALL_IMAGE_RENDERER_PLUGIN="false"

```

Comment:
You might want to set `ARG DEBIAN_FRONTEND=noninteractive` in order to ensure noninteractive installs.

Reply:
Good point